apps/wallet/src/ui/app/components/explorer-link/Explorer.ts

```diff
@@ -9,20 +9,23 @@ const API_ENV_TO_EXPLORER_URL: Record<API_ENV, string | undefined> = {
     [API_ENV.local]: process.env.EXPLORER_URL_LOCAL,
     [API_ENV.devNet]: process.env.EXPLORER_URL_DEV_NET,
     [API_ENV.staging]: process.env.EXPLORER_URL_STAGING,
+    [API_ENV.testNet]: process.env.EXPLORER_URL_TEST_NET,
+    //No explorer url for Custom PRC
+    [API_ENV.customRPC]: '',
```

Comment:
localhost:3000?

Reply:
I don't think we should include the explorer option unless we provide an input field for that. It will be confusing if the explorer link is NOT on `localhost:3000`

Reply:
Yeah that's fair, I'm okay with that.